### DISCUSSION TITLE
Opinions on New Project Boards

### DISCUSSION THREAD

**Discussion by JackEAllen** [community/events#23] [Community]
What does everyone think about new project boards? 
If you could have new features, what would they be?

**Comment by JackEAllen** [comment]
I quite like the new project boards but would like to see a few new features as it matures. 

If I have a project board with a custom grouping on a field such as milestones, I want to have the ability to select multiple issues and drag them into a grouping (e.g. add a load of newly created issues to a milestone or move many issues at once from a milestone to another milestone).

**Comment by queenofcorgis** [comment]
Hi @jackeallen,

Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we're glad you're here!

If you're looking to have a conversations and share features on Projects you might want to try asking in the main [GitHub Community](https://github.com/orgs/community/discussions/).

We want to make sure you’re getting the best support and eyes you can, but this space may not be the right place for this particular topic.

Best of luck!
